anime_downloader/gui/gui.py

```diff
@@ -0,0 +1,94 @@
+import eel
+from anime_downloader.sites import get_anime_class
+from anime_downloader.config import Config
+from anime_downloader import util
+from anime_downloader.sites.anime import AnimeEpisode
+import subprocess
+import pathlib
+
+@eel.expose
+def populate_dropdown():
+    from anime_downloader.sites import ALL_ANIME_SITES
+    providers = [i[1] for i in ALL_ANIME_SITES]
+    eel.populate_dropdown(providers)
+
+
+@eel.expose
+def anime_search(query, provider):
+    print('serching')
+    print(query, provider)
+    anime_cls = get_anime_class(provider)
+    print(anime_cls)
+    res = anime_cls.search(query)
+    print(res)
+
+    eel.populate_search([dict(title=i.title, url=i.url, poster=i.poster) for i in res])
+
+@eel.expose
+def get_animeinfo(url):
+    global anime
+    print('getting info of: '+ url)
+    anime = get_anime_class(url)(url)
+    print(anime)
+    print(anime[0].source().stream_url)
+    return {
+        'title': anime.title,
+        'length': len(anime),
+    }
+
+def play_mirror_file(player, title, file, ):
```

Comment:
why does this not call anime-dl's mpv function, that basically does the Popen along with some other stuff.

Reply:
Will fix, my bro was unaware of the utils in the project